### PR TITLE
Maciek zrob review

### PR DESCRIPTION
**Pull request by tojawawel**

### CODE REVIEW

File: app/jobs/update_tokens_job.rb [hunk 1]
```diff
@@ -1,9 +1,13 @@
+# frozen_string_literal: true
+
 class UpdateTokensJob < ApplicationJob
```
Comment (Maciorex) [inline app/jobs/update_tokens_job.rb:3]:
```suggestion
class UpdateTokenJob < ApplicationJob
```

I think we are updating just one token at time, am I right :)?

File: app/controllers/homes_controller.rb [hunk 2]
```diff
@@ -1,5 +1,8 @@
+# frozen_string_literal: true
+
 class HomesController < ApplicationController
-  before_action :set_home, only: [:show, :edit, :update, :destroy]
-  def index
-  end
+
+  before_action :set_home, only: %i[show edit update destroy]
```
Comment (Maciorex) [inline app/controllers/homes_controller.rb:5]: remove!

File: app/jobs/update_tokens_job.rb [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
+
   queue_as :default
 
   def perform
-    users = User.all 
+    users = User.all
     users.each { |user| AccessTokenRefresher.new.call(user) }
```
Comment (Maciorex) [inline app/jobs/update_tokens_job.rb:9]:
```suggestion
    User.all.each { |user| AccessTokenRefresher.new.call(user) }
```
For training purposes check out some methods to process users in batches. Maybe your app will have 1231242141 users soon :D

File: app/services/access_token_refresher.rb [hunk 4]
```diff
@@ -1,32 +1,41 @@
 # frozen_string_literal: true
+
 class AccessTokenRefresher
-  URL = "https://accounts.spotify.com/api/token"
 
-  def call(user)
-    token = new_access_token(user)
+  URL = 'https://accounts.spotify.com/api/token'
```
Comment (Maciorex) [inline app/services/access_token_refresher.rb:5]: I think that we should use `freeeze` on constants

File: app/services/get_playlist_list_service.rb [hunk 5]
```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class GetPlaylistListService
```
Comment (Maciorex) [inline app/services/get_playlist_list_service.rb:3]: `GetPlaylistListService` sounds like shit :D
`FetchListOfPlaylists` maybe?

Comment (tojawawel) [reply app/services/get_playlist_list_service.rb:3]: 👍

**Pull request merged**